mistralrs-server/src/main.rs

```diff
@@ -282,7 +282,7 @@ async fn main() -> Result<()> {
         warn!("Using flash attention with a quantized model has no effect!")
     }
     info!("Model kind is: {}", loader.get_kind().to_string());
-    let pipeline = loader.load_model(
+    let pipeline = loader.load_model_from_hf(
```

Comment:
If I understand correctly, all the logic within `get_paths!` is still there, so if a user specifies a local path, and they use this API, nothing will change?

Reply:
Correct. For example, take a look at https://github.com/EricLBuehler/mistral.rs/pull/308/files#diff-79ee2143b95e1682589015af457db1e09fe5d24b70501b22750104eb761084c3R350-R371